src/WingetCreateCLI/Commands/SubmitCommand.cs

```diff
@@ -83,7 +83,7 @@ private async Task<bool> SubmitManifest()
             {
                 List<string> manifestContents = Directory.GetFiles(this.Path).Select(f => File.ReadAllText(f)).ToList();
 
-                Serialization.DeserializeManifestContents(manifestContents, manifests);
+                manifests = Serialization.DeserializeManifestContents(manifestContents);
```

Comment:
[nit] manifests variable isn't shared outside if-scopes, so line 75 could be moved into the first if block, and declared anew here, that way we don't construct it unnecessarily

Reply:
Moved line 75 to first block and declare new Manifests object in 2nd block.